cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,103 @@
 /// <reference types='cypress' />
+const { generateWorker } = require('../support/generateWorker');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
 
+  beforeEach(() => {
+    cy.task('generateWorker').then((generateWorker) => {
+      worker = generateWorker;
+      cy.visit('/');
+    });
+  });
+
+  it('should check pagination', () => {
+    cy.get('.-pageInfo').should('exist');
+    cy.get('[aria-label="jump to page"]').type('1');
+  });
+
+  it('should check rows count selection', () => {
+    cy.get('[aria-label="rows per page"]').should('exist');
+    cy.get('[aria-label="rows per page"]').select('20 rows');
```

Comment:
assert here that number row have been changed

Reply:
Done!